server/src/main/java/org/elasticsearch/cluster/metadata/RollupMetadata.java

```diff
@@ -8,174 +8,10 @@
 
 package org.elasticsearch.cluster.metadata;
 
-import org.elasticsearch.Version;
-import org.elasticsearch.cluster.Diff;
-import org.elasticsearch.cluster.DiffableUtils;
-import org.elasticsearch.cluster.NamedDiff;
-import org.elasticsearch.common.ParseField;
-import org.elasticsearch.common.Strings;
-import org.elasticsearch.common.io.stream.StreamInput;
-import org.elasticsearch.common.io.stream.StreamOutput;
-import org.elasticsearch.common.xcontent.ConstructingObjectParser;
-import org.elasticsearch.common.xcontent.XContentBuilder;
-import org.elasticsearch.common.xcontent.XContentParser;
+public class RollupMetadata {
 
-import java.io.IOException;
-import java.util.EnumSet;
-import java.util.HashMap;
-import java.util.Map;
-import java.util.Objects;
-
-/**
- * Custom {@link Metadata} implementation for storing a map of {@link RollupGroup}s and their names.
- */
-public class RollupMetadata implements Metadata.Custom {
```

Comment:
Since this class is just holding static strings now. I do not think we need to have a dedicated class here in `org.elasticsearch.cluster.metadata`. What do you think if we brought in `TYPE`, `SOURCE_INDEX_NAME_META_FIELD`, `SOURCE_INDEX_ID_META_FIELD` into `RollupV2.java`?

Reply:
I had this in mind, but I wasn't sure where I should move those constants.